src/mscorlib/src/System/Collections/Hashtable.cs

```diff
@@ -66,6 +66,7 @@ namespace System.Collections
     // 
     [DebuggerTypeProxy(typeof(System.Collections.Hashtable.HashtableDebugView))]
     [DebuggerDisplay("Count = {Count}")]
+    [Serializable]
```

Comment:
Is there anything I need to watch out for when moving it from corefx to coreclr?